src/components/song-preview.js

```diff
@@ -7,24 +7,46 @@ const PREVIEW_VOLUME = 0.5;
  */
 AFRAME.registerComponent('song-preview-system', {
   schema: {
-    previewStartTime: {type: 'int'},
-    selectedChallengeId: {type: 'string'}
+    previewStartTime: { type: 'int' },
+    selectedChallengeId: { type: 'string' },
+    hash: { type: 'string' }
```

Comment:
what is this hash for?

Reply:
Not used right now, would only be needed to construct mp3 preview urls (when they support CORS). but maybe passing the url directly instead of the hash makes more sense

Reply:
yeah if we're going to pass URLs in other components it makes it more consistent